assignments/assignment01.rb

```diff
@@ -80,8 +101,11 @@ def my_reverse(s)
 # Write a method `palindrome?`
 # that determines whether a string is a palindrome
 def palindrome?(s)
-  stripped = s.delete(" ").delete(",").downcase
-  stripped == stripped.reverse
+
+  # strip out non alpha characters, downcase, check to see if the reverse is equivalent to the original string
+  # (must strip out non alpha characters to handle the last example case)
+  s.downcase.gsub(/[^a-z]/, '') == s.downcase.reverse.gsub(/[^a-z]/, '')
```

Comment:
you are executing the downcase and gsub twice. It would be more efficient to assign the result to a local variable, and then compare that variable to its reverse.